docs/photoniq/fingerprinting/index.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+title: Digital Fingerprinting
+---
+
+PhotonIQ Digital Fingerprinting is a managed service that generates persistent digital signatures and visitor IDs for your website users.
+
+## How It Works
+
+Digital Fingerprinting uses advanced fingerprinting techniques to create a unique signature of a user's device. This allows recognizing users across sessions and devices without relying only on cookies.
```

Comment:
"relying only on cookies" or "relying on cookies"?

Reply:
I thought we were using cookies, but not only cookies? But the description I found in Notion says we don't, so I guess the latter?

Reply:
We use cookies.  But if the cookie is not there, we have two other ways to identify them.

Now that I am reading it again, "relying only on cookies" works.

Sorry.

Reply:
@macrochrissie, might want to update your description in the PhotonIQ Launch. Fingerprinting does use cookies, but also other stuff.

Reply:
@joubert-macrometa, updated